### ISSUE TITLE
add license inside the program before releasing it!

### ISSUE THREAD

**Issue by em-mw** [ErMax-Inc/pic2ascii#9]
add license inside the program if we don't we will violate the license agreement when releasing the program.

**Comment by em-mw** [comment]
we should add this to both the gui and non-gui as we can release the non-gui version faster.

**Comment by em-et** [comment]
Removed the documentation label, as it doesn't have to do much with the docs.

**Comment by em-mw** [comment]
I did it!

**Comment by em-et** [comment]
You haven't made a pull request for it though... I saw the changes that you made, but you need to agree to the license first, then continue running the program. Viewing the license is nice too, but we need to implement the first one too, before releasing it.

**Comment by em-mw** [comment]
As of the "must do" in the licence, it doesn't specify to put an agreement in the program. Since it was not required I dicided not to put it in as we can focus more on the GUI release. Although I did not put it in you're more than welcome to edit it, or I can edit it for you.

**Comment by em-et** [comment]
No, it is good that it is in the program. I gave it (the license) a quick look and it seems as we have to include it inside the program. I do want to make a few changes though. If you can, revert back the program to it's original state, make a separate branch and pull request for the changes with the license.

**Comment by em-mw** [comment]
I have made the pull-request!

**Comment by em-et** [comment]
Thanks! I'll take a look at it!
